app/models/family_member.rb

```diff
@@ -19,41 +19,28 @@ class FamilyMember < ApplicationRecord
   # TODO: extract exception to its own file
   class FamilyKeyMismatch < StandardError
     def initialize(family)
-      super <<~MESSAGE
-        Attempted to create a family-bond between
-        #{family.person} and #{family.other},
-        but they seem to already to belong to different
-        families.
-
-        Both people have a "family_key", but not the same.
-
-        Data:
-        - Person: #{family.person_id}
-        - Other: #{family.other_id}
-        - Relation: #{family.kind} / #{family.kind_label}
-      MESSAGE
+      super('Attempted to create a family-bond between ' \
+        "#{family.person} and #{family.other}, " \
+        'but they seem to already to belong to different families. ' \
+        'Both people have a "family_key", but not the same. ' \
+        "Person: #{family.person_id}, other: #{family.other_id}, " \
+        "relation: #{family.kind} / #{family.kind_label}")
```

Comment:
Dies scheint noch ein Problem vom älteren Brakeman gewesen zu sein. Die aktuelle Version hat damit kein Problem. Können wir den Commit "Avoid syntax that brakeman does not understand" wieder herausnehmen?

Ich finde die Fehlermeldung deutlich lesbarer mit dem Heredoc. Notfalls könnte man es noch so anpassen:

```diff
- super <<~MESSAGE
+ super(<<~MESSAGE)
    Attempted to create a family-bond between
    ...
  MESSAGE
```

Reply:
Klappt super mit der neuen Version von Brakeman. Merci fürs ausprobieren. Es ist tatsächlich lesbarer so wie dus geschrieben hast.